tests/Doctrine/Tests/ORM/Mapping/AbstractMappingDriverTest.php

```diff
@@ -762,6 +762,10 @@ public function testSqlResultSetMapping(): void
      */
     public function testAssociationOverridesMapping(): void
     {
+        if (PHP_VERSION_ID >= 80000 && PHP_VERSION_ID < 80100) {
```

Comment:
This kind of condition is annoying, but required for some models we interact with. This skips all assocation/attribute override tests on PHP 8, but since we also run against 7.4 and 8.1 this is not a problem.